style.css

```diff
@@ -62,16 +94,367 @@
+.btns {
   display: flex;
-  flex-direction: column;
-  align-items: flex-start;
-  padding: 114px 24px;
-  background-image: url(image/header-shapes\ mobile.svg);
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  list-style: none;
+  padding-left: 0;
+}
+
+.view p {
+  padding: 10px;
+  border: solid black;
+  border-radius: 8px;
+  border-width: thin;
+  margin-bottom: 40px;
+  margin-top: auto;
+  border-color: #6070ff;
+}
+
+.connect-about p {
+  margin-bottom: 26px;
+  margin-top: 12px;
+}
+
+.view-about p {
+  padding: 10px;
+  border: solid black;
+  border-radius: 8px;
+  border-width: thin;
+  margin-bottom: 40px;
+  margin-top: auto;
+  margin-top: 36px;
+  border-color: #396df2;
+}
+
+.language p {
+  margin-left: 16px;
+}
+
+.snapshoot-5 p {
+  padding-left: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.frameworks p {
+  margin-left: 16px;
+}
+
+.btns li p {
+  margin-right: 32px;
+  text-decoration: none;
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin-top: auto;
+}
+
+.view {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+  margin: 0;
+  color: #396df2;
+}
+
+.view :hover {
+  background-color: #6070ff;
+  color: #fff;
+  border-color: #6070ff;
+}
+
+.view :active {
+  background-color: #2230d2;
+  color: #fff;
+  border-color: #2230d2;
+}
+
+.view :disabled {
+  background-color: #e5e5e5;
+  color: #5e6c84;
+  border-color: #5e6c84;
+}
+
+/* .Multi-Post-Stories-1 */
+
+.wrapper-3 {
+  width: 327px;
+  height: 596px;
+  padding: 16px;
+  margin: 0 24px;
+  margin-bottom: 88px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.snapshoot-2 {
   background-repeat: no-repeat;
+  background-image: url(image/Snapshoot\ Portfolio.svg);
+  min-height: 230px;
+  padding: 0 16px;
+}
+
+.multi-post-stories {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+}
+
+/* Tonic-2 */
+
+.snapshoot-3 {
+  background-image: url(image/Snapshoot\ 3Portfolio.svg);
+  background-repeat: no-repeat;
+  min-height: 230px;
+  padding: 0 16px;
+}
+
+/* Multi-Post-Stories-2 */
+
+.snapshoot-4 {
+  background-image: url(image/Snapshoot\ 4Portfolio.svg);
+  background-repeat: no-repeat;
+  min-height: 230px;
+  padding: 0 16px;
+}
+
+/* About Myself */
+
+.wrapper-about {
+  width: 300px;
+  height: 1263px;
+  padding: 24px;
+  margin-top: 114px;
+  padding-top: 114px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
```

Comment:
- [x]  Great job adding the `About` section to your project with all parts, but unfortunately your section is sticking to the left and not taking 100% width of the page as you can see in the screenshots below. Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. 
 - [x] As you can see in the screenshots below you need to have a right-top corner `border radius` in the `About` section, Kindly, refactor your code to be similar to the design

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/149560838-1df80e8f-3b1e-485e-a3e3-71cdf67264ed.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/149560833-a5daa8b5-1493-4909-bbf4-85b46921bee7.PNG)